SpectraLogic.SpectraRioBrokerClient/Model/Job.cs

```diff
@@ -36,19 +36,21 @@ public class Job : IJob
         /// <param name="filesTransferred">The files transferred.</param>
         /// <param name="totalSizeInBytes">The total size in bytes.</param>
         /// <param name="bytesTransferred">The bytes transferred.</param>
-        /// <param name="created">The created.</param>
+        /// <param name="creationDate">The created.</param>
```

Comment:
not a sentence. Change to something like: `The creation date` or `The date of creation`